2-multitenant/README.md

```diff
@@ -1,2 +1,54 @@
 # Multitenant Infrastructure phase
-This phase deploys multitenant infrastruture using the multitenant infrastructure pipeline.
+
+## Purpose
+
+This phase deploys the per-environment multitenant resources deployed via the multitenant infrastructure pipeline.
+
+The following resources are created:
+- GCP Projects
```

Comment:
should it be singular?
If not, I'd mention what is the cardinality of projects?